apps/wing/src/wingc.ts

```diff
@@ -1,15 +1,12 @@
 import debug from "debug";
-import { readFile } from "fs/promises";
-import { resolve } from "path";
-import { WASI } from "wasi";
+import { readFileSync } from "fs";
 import { normalPath } from "./util";
+import WASI from "wasi-js";
```

Comment:
(checking my understanding) So unlike the `wasi` module which is a node builtin, `wasi-js` works both browser-side and server-side? Big if true

Reply:
Yessir